add feature availability to embed pages (#1082)
n8n-io/n8n-docs: _snippets/embed-license.md

!!! info "Feature availability"

> 		Embed requires an embed license. For more information about when to use Embed, as well as costs and licensing processes, refer to [Embed](https://n8n.io/embed/){:target=_blank .external-link} on the n8n website..

Author: ```suggestion
		Embed requires an embed license. For more information about when to use Embed, as well as costs and licensing processes, refer to [Embed](https://n8n.io/embed/){:target=_blank .external-link} on the n8n website.
```

> 		Embed requires an embed license. For more information about when to use Embed, as well as costs and licensing processes, contact n8n using [this form](https://n8n-community.typeform.com/to/qOBgyWWA){:target=_blank .external-link}.

Author: ```suggestion
		Embed requires an embed license. For more information about when to use Embed, as well as costs and licensing processes, refer to [Embed](https://n8n.io/embed/){:target=_blank .external-link} on the n8n website..
```
